src/styles/_App.scss

```diff
@@ -0,0 +1,9 @@
+
+body {
+  margin: 0;
+}
+
+.app {
+  width: 100vw;
```

Comment:
it might be useful to go ahead and set a max-width here to avoid the huge screen dilemma